### ISSUE TITLE
KeyError at --as step

### ISSUE THREAD

**Issue by pieterlukasse** [opentargets/platform#491]
Spotted this error in the --as step:

```
2019-03-13 11:05:24,219 mrtarget.common.Redis DEBUG - ========> Processing evidence: target[ENSG00000185527], disease[EFO_0000311]
2019-03-13 11:05:24,279 mrtarget.common.ElasticsearchLoader WARNING - push to elasticsearch failed for chunk.  retrying in 5s...
2019-03-13 11:05:24,668 mrtarget.common.Redis ERROR - Error processing job 1870319318ba412aa8d5f94c44cf4d15: hits
Traceback (most recent call last):
  File "mrtarget/common/Redis.py", line 300, in _inner_run
    job_results = self.process(data)
  File "mrtarget/modules/Association.py", line 329, in process
    available_evidence = self.es_query.count_evidence_for_target(target)
  File "mrtarget/common/ElasticsearchQuery.py", line 451, in count_evidence_for_target
    return res['hits']['total']
KeyError: 'hits'
2019-03-13 11:05:24,696 mrtarget.common.Redis DEBUG - ############# found 64 evidences for target ENSG00000113721 
2019-03-13 11:05:24,821 mrtarget.common.Redis DEBUG - ############# found 1 evidences for target ENSG00000154678 
```

**Comment by afaulconbridge** [comment]
I wonder if this is a similar problem to the file descriptor `fork() issue descibed in opentargets/platform#493

For 19.04 the redis based queues will be replaced with pypeln . I will have a close look at our log files to see if this has happened recently.

**Comment by afaulconbridge** [comment]
I've reviewed our most recent internal pipeline runs and seen no evidence of this problem. Therefore I expect it to be resolved with 19.04. I'll leave this ticket open for now, and it should be re-examined with 19.04 to ensure the problem is solved.

**Comment by pieterlukasse** [comment]
seems to be solved in 19.04 :+1:
